example/thirdparty/netty/build.mill

```diff
@@ -636,16 +638,18 @@ object `transport-udt` extends NettyModule {
 
 > ./mill -j5 __.compile
 
-> ./mill 'codec-{dns,haproxy,http,http2,memcache,mqtt,redis,smtp,socks,stomp,xml}.test'
+> ./mill 'codec-stomp.test.testForked'
```

Comment:
I think the idea was that all tests must pass. This is now no longer checked.

Reply:
It is still checked, right after this line. The diff doesn't make it obvious, but the test coverage shouldn't be lower.

Reply:
Thanks for pointing this out.